weaver/core/drivers/fabric-driver/server/satp.ts

```diff
@@ -0,0 +1,363 @@
+import satp_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_pb';
+import satp_grpc_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_grpc_pb';
+import driverPb from '@hyperledger/cacti-weaver-protos-js/driver/driver_pb';
+import logger from './logger';
+import { credentials } from '@grpc/grpc-js';
+import { SatpAssetManager, HashFunctions } from '@hyperledger/cacti-weaver-sdk-fabric'
+import fs from 'fs';
+import path from 'path';
+import { Gateway, Network, Wallets, Contract, X509Identity } from 'fabric-network'
+import { getNetworkGateway } from "./fabric-code";
+import { getDriverKeyCert } from './walletSetup';
+
+const DB_NAME: string = "driverdb";
+const DRIVER_ERROR_CONSTANTS = JSON.parse(
+    fs.readFileSync(
+        path.resolve(__dirname, '../constants/driver-error-constants.json'),
+    ).toString(),
+);
+
+const getWallet = (walletPath: string) => {
+    return Wallets.newFileSystemWallet(walletPath);
+};
+
+async function performLockHelper(
+    performLockRequest: driverPb.PerformLockRequest,
+    networkName: string
+): Promise<any> {
+
+    // TODO: remove the hardcoded values 
```

Comment:
When will the hardcodes be removed? (applies to entire PR)

Reply:
@outSH This entire file consists of sample functions designed to test the SATP protocol end-to-end. I think the best way to fix this for now is to rename the source file to `sample-satp.ts`.

@outsidethecode Can you rename the file and make sure the references to this file in the `fabric-driver` folder are updated wherever necessary?

Reply:
Renamed as recommended.